src/common/com/intellij/plugins/haxe/util/HaxePsiUtils.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
I have moved all methods to UsefulPsiTreeUtil.

By then I didn't want to congest the namespace. And probably I though it was related to recursive stuff. But moving static methods is an easy and fast refactor so it was not a big deal.